.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  env: {
+    browser: true,
+    commonjs: true,
+    es2021: true
+  },
+  extends: 'standard',
```

Comment:
Could we stick with something closer to what we use at Sentry? See https://github.com/getsentry/sentry/blob/d92c8f42227ab9ff4e834b1e063bdabdce28e8b4/prettier.config.js